src/main/java/medicaldoctor/controllers/LookUp.java

```diff
@@ -22,7 +22,7 @@
     public static final String PATIENT_LOOKUP = "PatientLookup";
     public static final String DOCTOR_LOOKUP = "DoctorLookup";
     public static final String USER_LOOKUP = "UserLookup";
-    public static final String MY_PATIENT = "MyPatients";
+    public static final String MY_PATIENT = "MyPatient";
```

Comment:
why no 's', doctors have plural patients

Reply:
I'll make that change.